docs/Zendesk-batchsource.md

```diff
@@ -77,16 +79,24 @@ corresponding CDAP types.
 | Array                  | Array     |
 | Record                 | Record    |
 
-Data Duplication issues with Zendesk APIs
+
+Limitations
 ----------
-This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/ticketing/managing-tickets/using-the-incremental-export-api/#time-based-incremental-exports)  pagination.
```

Comment:
looks like we removed Cursor Pagination: Results in Data Loss
but didn't change Zendesk supports three types of... to "two types of"

Reply:
Can you delete the comma before time-based and just say "offset and time-based"

Reply:
done